packages/c/csv2/xmake.lua

```diff
@@ -0,0 +1,14 @@
+package("csv2")
+    set_urls("https://github.com/p-ranav/csv2.git")
+    set_homepage("https://github.com/p-ranav/csv2")
+    set_description("A CSV parser library")
+    set_license("MIT")
+    add_versions("v0.1", "a20992f7a1b75215609037c4cc66f690e8691aab")
```

Comment:
`a20992f7a1b75215609037c4cc66f690e8691aab ` it should be the shasum256 of tar.gz